src/helpers/inStockTv.js

```diff
@@ -0,0 +1,10 @@
+
+let inStockCounter = 0;
+function inStockTv (){
+    for (let i = 0; i < inventory.originalStock.length; i++) {
+        inStockCounter = inStockCounter + inventory.originalStock[i];
```

Comment:
Hier zet je de index nummer op originalStock, maar dit is een integer en niet de entry van een array. Je moet dus eigenlijk dit doen: inventory[i].originalStock